README.md

```diff
@@ -711,6 +712,36 @@ end
 
 ---
 
+### Adding a string column
+
+In Postgres, there is no performance benefit to using the `:string` (`varchar(255)`) type over `:text`, and `:string` can cause failures when inserting data if the value is longer than 255 characters.
```

Comment:
IIRC it's actually slightly slower to use `varchar(n)` because of an additional constraint check. This might be outdated info though.